### PR TITLE
New param in service-environment-costs endpoint: forecast

### PR DESCRIPTION
**Pull request by mkurek**
Set it to true to fetch forecast costs (by default "regular" costs are returned)

### CODE REVIEW

File: src/ralph_scrooge/rest/service_environment_costs.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
 )
 
 
-@memoize
+# @memoize
```
Comment (xor-xor) [inline src/ralph_scrooge/rest/service_environment_costs.py]: Why is that commented out..?

File: src/ralph_scrooge/tests/rest/test_service_environment_costs.py [hunk 2]
```diff
@@ -284,6 +286,37 @@ def test_if_grouped_date_has_correct_format_when_group_by_month_given(self):  #
         date_received = json.loads(resp.content)['service_environment_costs'][0]['grouped_date']  # noqa: E501
         self.assertEquals(date_received, date_expected)
 
+    def test_for_forecast_costs(self):  # noqa: E501
```
Comment (xor-xor) [inline src/ralph_scrooge/tests/rest/test_service_environment_costs.py]: `noqa` is not needed here, I guess..?

File: src/ralph_scrooge/utils/common.py [hunk 3]
```diff
@@ -6,10 +6,10 @@
 from __future__ import unicode_literals
 
 from decimal import Decimal
-from functools import wraps
 
 from django.conf import settings
-from ralph_scrooge.utils.cache import memoize as memoize_orig
+# DEPRECATED. Import memoize directly from `ralph_scrooge.cache`
+from ralph_scrooge.utils.cache import memoize  # noqa: F401
```
Comment (xor-xor) [inline src/ralph_scrooge/utils/common.py:12]: So... why do we need this import here..?

**Review by xor-xor — CHANGES_REQUESTED** [review]

File: src/ralph_scrooge/rest/service_environment_costs.py (hunk 1 above)
Comment (mkurek) [reply src/ralph_scrooge/rest/service_environment_costs.py]: right - I should revert it

File: src/ralph_scrooge/utils/common.py (hunk 3 above)
Comment (mkurek) [reply src/ralph_scrooge/utils/common.py:12]: because still lot of modules import `memoize` from there

Comment (xor-xor) [reply src/ralph_scrooge/utils/common.py:12]: OK.

**Review by xor-xor — APPROVED** [review]

**Review by szok — APPROVED** [review]

**Pull request merged**